TerminalDocs/command-palette.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Windows Terminal Command Palette
+description: Learn how to use the command palette in the Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 08/26/2020
+ms.topic: how-to 
+ms.service: terminal
+---
+
+# How to use the command palette in Windows Terminal ([Preview](https://aka.ms/terminal-preview/))
+
+The command palette lets you see which commands you can run inside Windows Terminal.
+
+## Invoking the command palette
+
+You can invoke the command palette by typing `ctrl+shift+p`. This can be customized by adding the `commandPalette` command to your key bindings.
+
+```json
+{ "command": "commandPalette", "keys": "ctrl+shift+p" }
+```
+
+## Command line mode
+
+If you'd like to enter a `wt` command into the command palette, you can do so by starting with the `>` character. This will run the `wt` command in the current window.
+
+## Defining commands
+
+There are three different ways you can define a command in Windows Terminal.
+
+1. With a value for `keys` without a `name`
+
+This is the behavior that is documented on the [key bindings page](./customize-settings/key-bindings.md). These commands can be invoked with the defined key binding. The commands will appear in the command palette if Windows Terminal can autogenerate a reasonable name for it.
+
+```json
+{ "command": "newTab", "keys": "ctrl+shift+t" }
+```
+
+2. Without a value for `keys`, but with a `name`
+
+These commands will only be added to the command palette. They cannot be invoked with a key binding. `name` overrides the autogenerated name.
+
+```json
+{ "name": "Open my default tab", "action": "newTab" }
+```
+
+3. With a value for `keys` with a `name`
+
+These commands will be added to the command palette and can be invoked with the keyboard. The `name` property will override the autogenerated name.
+
+```json
+{ "name": "New tab", "command": "newTab", "keys": "ctrl+shift+t" }
+```
+
+### Adding an icon
+
+You can optionally add an icon to a command that appears in the command palette. This can be done by adding the `icon` property to the command.
+
+```json
+{ "icon": "", "name": "New tab", "command": "newTab", "keys": "ctrl+shift+t" }
```

Comment:
We should note all the possible formats that an icon can take:https://github.com/microsoft/terminal/pull/7368

* The icon can be a path to an image (don't forget to escape the backslashes!)
* The icon can be a symbol from  [Segoe MDL2 Assets](https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font), using the syntax `"\uE8A3"`
* The icon can be any character, including emoji. For example, `"⚡"`